feat(cdn): CDN domain support new fields (#4622)
huaweicloud/terraform-provider-huaweicloud: docs/resources/cdn_domain.md

  **type_c1**: Method C1.
  **type_c2**: Method C2.

* `sign_method` - (Optional, String) Specifies the encryption algorithm type for URL authentication.
  The valid values are as following:
  + **md5**

>   + **sha256**

Reviewer: Please note its default value.

* `match_type` - (Optional, String) Specifies the authentication scope.

>   Currently, only spuuort value is **all**, indicates that all files are involved in authentication.

Reviewer: The word is `support`, and there's something wrong with its syntax.

Reviewer: All optional values ​​need to add default value description, please check all documents.

* `sign_arg` - (Optional, String) Specifies the authentication parameters.
  The valid length is limited from `1` to `100` characters, only letters, digits, and underscores (_) are allowed.

>   The value cang not start with a digit.

Reviewer: Please  check all words in document. The correct word is `can`.

* `inherit_config` - (Optional, List) Specifies the details of the authentication inheritance.
  The [inherit_config](#inherit_config_object) structure is documented below.

> * `key` - (Optional, String) Specifies the authentication key contains `16` to `32` characters, including letters and digits.

Reviewer: This field is required when enable URL signing.
Please refer to the console to complete the required scenes.

* `backup_key` - (Optional, String) Specifies the standby authentication key contains `16` to `32` characters,
  including letters and digits.

* `time_format` - (Optional, String) Specifies the time format. Possible values are:
  **dec**: Decimal, can be used in Method A, Method B and Method C2.
  **hex**: Hexadecimal, can be used in Method C1 and Method C2.

* `expire_time` - (Optional, Int) Specifies the expiration time. The value ranges from `0` to `31536000`, in seconds.

<a name="inherit_config_object"></a>
The `inherit_config` blocks support:

* `enabled` - (Required, Bool) Specifies whether to enable authentication inheritance.

> * `inherit_type` - (Optional, String) Specifies the authentication inheritance configuration.

Reviewer: ```suggestion
* `inherit_type` - (Optional, String) Specifies the authentication inheritance configuration. The valid values are **m3u8** and **mpd**. Separate multiple values with commas (,). e.g. **m3u8,mpd**.
```

  The valid values are as follows:
  + **m3u8**
  + **mpd**

-> Separate multiple values with commas (,). e.g. **m3u8,mpd**.

> -> This parameter is mandatory when authentication inheritance is enabled.

Reviewer: ```suggestion
  -> This parameter is mandatory when authentication inheritance is enabled.
```

> * `inherit_time_type` - (Optional, String) Specifies the start time of authentication inheritance.

Reviewer: ```suggestion
* `inherit_time_type` - (Optional, String) Specifies the time type that inherits authentication settings.
```

  The valid values are as follows:
  + **sys_time**: The current system time.

>   + **parent_url_time**: The time consistent with the M3U8 and MPD access links.

Reviewer: ```suggestion
  + **parent_url_time**: The time when a user accesses the M3U8/MPD file.
```
